docs: rephrase dockerize section (#4298)
jina-ai/jina: docs/fundamentals/executor/containerize-executor.md

## Prerequisites

To be able to understand how a Container image is built for an Executor, you need an understanding of [docker](https://docs.docker.com/) generally, of how to write 
a [Dockerfile](https://docs.docker.com/engine/reference/builder/), and how to build a docker image.

Also, to reproduce the example below it is required to have `docker` installed locally.

## Installing Jina in the Docker image

It is strictly needed for Jina to be installed inside the docker image. This can be achieved in 2 ways:

- Use a [Jina based image](https://hub.docker.com/r/jinaai/jina) as the base image in your Dockerfile.
This will make sure that everything needed for Jina to run the Executor is installed.

```dockerfile
FROM jinaai/jina:3.0-py37-perf
```

- Simply install Jina like any other Python package. You can do this by specifying Jina in the `requirements.txt`, 
or by including the `pip install jina` command as part of the image building process.  

```dockerfile
RUN pip install jina==3.0
```

## Setting Jina Executor CLI as entrypoint

When a containerized Executor is run inside a Flow,
under the hood Jina basically executes `docker run` with some extra arguments.

> This means that Jina assumes that what runs inside the `container` is the same as it would run in a regular OS process. Therefore, you need to make sure that

Author: I am not sure what this sentence means (again, I'm still a docker idiot). Would this be a fair rephrasing?

```suggestion
This means that Jina assumes that what runs inside the `container` runs just like it would run in a regular OS process. Therefore, you need to make sure that
```

Reviewer: Yes it would be correct
